2023-06-22 17:55:28,032 INFO    Thread-2  :221605 [wandb_setup.py:_flush():76] Current SDK version is 0.15.4
2023-06-22 17:55:28,032 INFO    Thread-2  :221605 [wandb_setup.py:_flush():76] Configure stats pid to 221605
2023-06-22 17:55:28,032 INFO    Thread-2  :221605 [wandb_setup.py:_flush():76] Loading settings from /home/min/a/jgammell/.config/wandb/settings
2023-06-22 17:55:28,032 INFO    Thread-2  :221605 [wandb_setup.py:_flush():76] Loading settings from /local/a/jgammell/dlsca_bag_of_tricks/src/wandb/settings
2023-06-22 17:55:28,033 INFO    Thread-2  :221605 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'htune__zaidnet_big__toy_aes', 'entity': 'jgammell', 'root_dir': '/local/a/jgammell/dlsca_bag_of_tricks/src', 'sweep_id': 'qbou2xxh', 'run_id': 'hzv0hvm8', 'sweep_param_path': '/local/a/jgammell/dlsca_bag_of_tricks/src/wandb/sweep-qbou2xxh/config-hzv0hvm8.yaml'}
2023-06-22 17:55:28,033 INFO    Thread-2  :221605 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-22 17:55:28,033 INFO    Thread-2  :221605 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'src/main.py', 'program': '/local/a/jgammell/dlsca_bag_of_tricks/src/main.py'}
2023-06-22 17:55:28,033 INFO    Thread-2  :221605 [wandb_init.py:_log_setup():507] Logging user logs to /local/a/jgammell/dlsca_bag_of_tricks/src/wandb/run-20230622_175528-hzv0hvm8/logs/debug.log
2023-06-22 17:55:28,033 INFO    Thread-2  :221605 [wandb_init.py:_log_setup():508] Logging internal logs to /local/a/jgammell/dlsca_bag_of_tricks/src/wandb/run-20230622_175528-hzv0hvm8/logs/debug-internal.log
2023-06-22 17:55:28,033 INFO    Thread-2  :221605 [wandb_init.py:init():547] calling init triggers
2023-06-22 17:55:28,034 INFO    Thread-2  :221605 [wandb_init.py:init():554] wandb.init called with sweep_config: {'pooling_method': 'aa', 'scheduler_kwargs-max_lr': 0.003461757787823417}
config: {}
2023-06-22 17:55:28,034 INFO    Thread-2  :221605 [wandb_init.py:init():596] starting backend
2023-06-22 17:55:28,034 INFO    Thread-2  :221605 [wandb_init.py:init():600] setting up manager
2023-06-22 17:55:28,048 INFO    Thread-2  :221605 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-06-22 17:55:28,059 INFO    Thread-2  :221605 [wandb_init.py:init():606] backend started and connected
2023-06-22 17:55:28,068 INFO    Thread-2  :221605 [wandb_run.py:_config_callback():1283] config_cb None None {'pooling_method': 'aa', 'scheduler_kwargs-max_lr': 0.003461757787823417}
2023-06-22 17:55:28,073 INFO    Thread-2  :221605 [wandb_init.py:init():703] updated telemetry
2023-06-22 17:55:28,112 INFO    Thread-2  :221605 [wandb_init.py:init():736] communicating run to backend with 60.0 second timeout
2023-06-22 17:55:28,391 INFO    Thread-2  :221605 [wandb_run.py:_on_init():2176] communicating current version
2023-06-22 17:55:28,540 INFO    Thread-2  :221605 [wandb_run.py:_on_init():2185] got version response 
2023-06-22 17:55:28,540 INFO    Thread-2  :221605 [wandb_init.py:init():787] starting run threads in backend
2023-06-22 17:55:34,062 INFO    Thread-2  :221605 [wandb_run.py:_console_start():2155] atexit reg
2023-06-22 17:55:34,062 INFO    Thread-2  :221605 [wandb_run.py:_redirect():2010] redirect: SettingsConsole.WRAP_RAW
2023-06-22 17:55:34,064 INFO    Thread-2  :221605 [wandb_run.py:_redirect():2075] Wrapping output streams.
2023-06-22 17:55:34,064 INFO    Thread-2  :221605 [wandb_run.py:_redirect():2100] Redirects installed.
2023-06-22 17:55:34,065 INFO    Thread-2  :221605 [wandb_init.py:init():828] run started, returning control to user process
2023-06-22 17:55:34,068 INFO    Thread-2  :221605 [wandb_run.py:_finish():1890] finishing run jgammell/htune__zaidnet_big__toy_aes/hzv0hvm8
2023-06-22 17:55:34,068 INFO    Thread-2  :221605 [wandb_run.py:_atexit_cleanup():2124] got exitcode: 1
2023-06-22 17:55:34,068 INFO    Thread-2  :221605 [wandb_run.py:_restore():2107] restore
2023-06-22 17:55:34,069 INFO    Thread-2  :221605 [wandb_run.py:_restore():2113] restore done
